packages/measure-tools/src/api/FeatureTracking.ts

```diff
@@ -1,6 +1,6 @@
 /*---------------------------------------------------------------------------------------------
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
-* See LICENSE.md in the project root for license terms and full copyright notice.
+* See COPYRIGHT.md in the repository root for full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 
 import { BeUiEvent, GuidString, Logger } from "@bentley/bentleyjs-core";
```

Comment:
hate that this exists, we should chat about this later

Reply:
lol yea, I'll clean all this up in a separate PR.